content/apps/production-ready.md

```diff
@@ -2,54 +2,84 @@
 
-### How
+#### How
+* Use the `cf marketplace` to consider your options and sizes for each service and choose appropriately for the resources your application will need.
 
-See the [multiple instances]({{< relref "multiple-instances.md" >}}) page.
+### Space per environment
+When running an application for development or testing, it is best to have a separate space from your production instance in which to run the application.
 
-## Prefer User Provided Services
+#### How
+* As an Org Manager for your organization, use the `cf create-space` command to create new spaces for each environment
 
-Prefer user provided services over environment variables for a secure centralized way to store application credentials and variables.
+### Health monitoring
+You want to be able to receive alerts about application errors, downtime, and throughput issues.
 
-### How
+#### How
+* NewRelic provides uptime monitoring with "Insights". It is easy to set up and
+receive alerts on a variety of metrics.
 
-Create user provided services with `cf cups` bind them with `cf bs`.
+## Additional practices
+The following practices are very helpful to incorporate into most cloud.gov apps. Check these out and evaluate which ones you need, depending on the needs of your team and users.
 
-## NewRelic (or similar)
+### Zero-Downtime deploy
+Your application should be able to be deployed without generating any downtime. Be aware there are known issues if your application automatically does database migrations when deploying.
 
-You want to be able to receive alerts about application errors and throughput issues.
+#### How
+* Use the [autopilot](https://github.com/concourse/autopilot) Cloud Foundry CLI plugin or the [`cf-blue-green`](https://github.com/18F/cf-blue-green) tool; see instructions in the respective repositories.
 
-## Uptime monitor
+### Automated deployments
+To reduce the risk associated with manual deployments, consider automating the process so it is repeatable.
 
-NewRelic provides uptime monitoring with "Insights". It is easy to set up and
-receive alerts for downtime.
+#### How
+* See the [continuous deployment]({{< relref "continuous-deployment.md" >}}) page.
 
-## Caching
+### Caching
+The best way to prevent performance issues is by having caching enabled on your application.
 
-The best way to prevent performance issues is by having caching enabled on your
-application.
-Cloud.gov has a memcached service but you can also rely on S3 or file storage for caches.
+#### How
+* Cloud.gov has a memcached service but you can also rely on S3 or file storage for caches.
 
-## Asset Serving
+### Asset Serving
+It is best not to serve static files from Cloud.gov directly.
 
-It is best not to serve static files from Cloud.gov directly. You can store your
-files in S3 or point CloudFront to an assets folder so you serve your assets with a CDN.
+#### How
+* You can store your files in S3 or point CloudFront to an assets folder so you serve your assets with a CDN.
```

Comment:
I agree with @afeld 's sentiment: avoid the extra complexity of an asset server, and let the CDN optimize it.